python/src/aac/spec/spec.yaml

```diff
@@ -340,6 +340,9 @@ schema:
         - name
         - fields
     - name: Unique definition names
+    - name: Definition references exist
+      arguments:
+        - name
```

Comment:
In my comment above I touch on how that the suggested edit for the validation implementation is not leveraging arguments as we rely exclusively on definition names to identify references between definitions.
```suggestion
```